diff --git a/node_modules/expo-font/build/ExpoFontLoader.web.js b/node_modules/expo-font/build/ExpoFontLoader.web.js
index be6b8c8..51c763d 100644
--- a/node_modules/expo-font/build/ExpoFontLoader.web.js
+++ b/node_modules/expo-font/build/ExpoFontLoader.web.js
@@ -134,7 +134,7 @@ export default {
         return new FontObserver(fontFamilyName, {
             // @ts-expect-error: TODO(@kitten): Typings indicate that the polyfill may not support this?
             display: resource.display,
-        }).load(null, 6000);
+        }).load(null, 90000);
     },
 };
 const ID = 'expo-generated-fonts';
